### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by aaryangrg**
IMPORTANT NOTES (please read and then delete):

* The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Changed UI layout from horizontal to vertical.".

* Please make sure to mention "#bugnum" somewhere in the description of the PR. This enables Github to link the PR to the corresponding bug.

Please also make sure to follow the [contribution](https://github.com/Cloud-CV/Fabrik/blob/master/.github/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,38 @@
...
-This app is presently under active development and we welcome contributions. Please check out our [issues thread](https://github.com/Cloud-CV/IDE/issues) to find things to work on, or ping us on [Gitter](https://gitter.im/Cloud-CV/IDE).
+This app is presently under active development and welcomes contributions. Please check out our [issues thread](https://github.com/Cloud-CV/IDE/issues) to find things to work on, or ping us on [Gitter](https://gitter.im/Cloud-CV/IDE).
 
 
 ## Installation Instructions
 
-Setting up Fabrik on your local machine is really easy. You can setup Fabrik using two methods:
+Setting up Fabrik on your local machine is very easy. You can setup Fabrik using two methods:
 
 ### Using Docker
```
Comment (Ram81) [inline README.md]: I don't think we need to add installation instructions for docker here, you can just add a reference from where anyone can install docker. Our README should be focused on setting up Fabrik. Can you remove this.

File: README.md [hunk 2]
```diff
@@ -4,19 +4,38 @@
...
+Fabrik is an online collaborative platform which allows one to build, visualize and train deep learning models via a simple drag-and-drop interface. It allows researchers to collectively develop and debug models using a web GUI which supports importing, editing and exporting networks written using popular frameworks like Caffe, Keras, and TensorFlow.
 
 <img src="/example/fabrik_demo.gif?raw=true">
 
-This app is presently under active development and we welcome contributions. Please check out our [issues thread](https://github.com/Cloud-CV/IDE/issues) to find things to work on, or ping us on [Gitter](https://gitter.im/Cloud-CV/IDE).
+This app is presently under active development and welcomes contributions. Please check out our [issues thread](https://github.com/Cloud-CV/IDE/issues) to find things to work on, or ping us on [Gitter](https://gitter.im/Cloud-CV/IDE).
```
Comment (Ram81) [inline README.md]: Please revert this line.

File: README.md (hunk 1 above)
Comment (aaryangrg) [reply README.md]: yes I agree . I was asked to add It . will incorporate all the changes in the next commit . sorry
thanks!

File: README.md [hunk 3]
```diff
@@ -13,9 +13,12 @@ This app is presently under active development and we welcome contributions. Ple
 
 ## Installation Instructions
 
-Setting up Fabrik on your local machine is really easy. You can setup Fabrik using two methods:
+Setting up Fabrik on your local machine is very easy. You can setup Fabrik using two methods:
 
 ### Using Docker
+If you haven't installed Docker already: 
+[Use this link](https://docs.docker.com/docker-for-windows/install/) if you are using Windows.
```
Comment (Ram81) [inline README.md]: Can you make this as two separate lines, doesn't look good on markdown

**Comment by aaryangrg** [comment]
Okay will do . Thanks

File: README.md [hunk 4]
```diff
@@ -13,9 +13,12 @@ This app is presently under active development and we welcome contributions. Ple
 
 ## Installation Instructions
 
-Setting up Fabrik on your local machine is really easy. You can setup Fabrik using two methods:
+Setting up Fabrik on your local machine is very easy. You can setup Fabrik using two methods:
 
 ### Using Docker
+If you haven't installed Docker already: </br>
+[Use this link](https://docs.docker.com/docker-for-windows/install/) if you are using Windows. </br>
+[Use this link](https://docs.docker.com/docker-for-mac/install/) if you are using a Mac.
```
Comment (Ram81) [inline README.md]: @aaryangrg It still doesn't look neat, can you make it a little bit clean. You can refer any other lines from readme to make it clean.

**Comment by aaryangrg** [comment]
Okay sure . I need to learn how to do somethings will get back to you in
sometime

**Pull request merged**